gw_eccentricity/load_data.py

```diff
@@ -835,7 +835,8 @@ def load_sxs_catalogformat(**kwargs):
                 "used to eavaluate the zero ecc waveform.")
 
     # Check file format
-    if "Strain_N" in kwargs["filepath"]:
+    matches = re.findall(r"Strain_N\d.h5", kwargs["filepath"])
+    if len(matches) == 1:
```

Comment:
This way you don't catch cases where len(matches) != 1, right? You need an Exception. Also, hopefully you are testing this code?

Reply:
Done